lib/Minz/Request.php

```diff
@@ -95,18 +96,20 @@ public static function _param($key, $value = false) {
 	 */
 	public static function init() {
 		self::magicQuotesOff();
+		static::$headers = static::getHeaders();
 		self::initJSON();
 	}
 
 	/**
 	 * Return true if the request is over HTTPS, false otherwise (HTTP)
+	 *
+	 * @return boolean
 	 */
 	public static function isHttps() {
-		if (isset($_SERVER['HTTP_X_FORWARDED_PROTO'])) {
-			return strtolower($_SERVER['HTTP_X_FORWARDED_PROTO']) === 'https';
-		} else {
-			return isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] === 'on';
+		if (null !== $header = static::getHeader('HTTP_X_FORWARDED_PROTO')) {
```

Comment:
Nitpick: I prefer to assign the variable on its own line, I think it makes the code more readable for newcomers. Just my opinion :) Looking at the rest of the code, I understand you preferred this solution anyway!

Reply:
Not just for newcomers, it's just clearer at a glance while skimming.

Reply:
Here there is only one variable so it does not make any difference. I change it back. But later in the code (`extractHost` function) it will be unreadable to have multiple assignments. I think that in that case, the variable has such a limited scope that it makes no sense to assign it separately.